bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionParameterGroup.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.config.core;
+
+/**
+ * The {@link ConfigDescriptionParameterGroup} specifies information about parameter groups.
+ * A parameter group is used to group a number of parameters together so they can
+ * be displayed together in the UI (eg in a single tab).
+ * <p>
+ * A {@link ConfigDescriptionParameter} instance must also contain the groupId. It should be permissible to use the
+ * groupId in the {@link ConfigDesctiptionParameter} without supplying a corresponding
+ * {@link ConfigDescriptionParameterGroup} - in this way the UI can group the parameters together, but doesn't have the
+ * group information.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class ConfigDescriptionParameterGroup {
+
+    private String name;
+    private String context;

```

Comment:
The context for parameters doesn't need to be the same as groups. I'd like to be able to provide some sort of context for this group to the user. In ZWave, there are a number of distinctive sets of data types that will be grouped together and I think it's useful to be able to indicate what 'type' they are.
Maybe context isn't the right term - I'm happy to change the name...